client/src/pages/private/CohortDetails.js

```diff
@@ -103,14 +103,14 @@ export default ({ match }) => {
               <Grid item xs={12} sm={6}>
                 <Typography variant='subtitle2'>Start Date</Typography>
                 <Typography variant='body1'>
-                  {dayjs(cohort?.start_date).format('MMM DD, YYYY')}
+                  {dayjs.utc(cohort?.start_date).format('MMM DD, YYYY')}
```

Comment:
Oh, interesting... That tells me that we store everything in UTC in database. Right? Awesome